src/commands/build/wranglerjs/output.rs

```diff
@@ -24,50 +25,87 @@ impl WranglerjsOutput {
         self.errors.join("\n")
     }
 
-    pub fn script_size(&self) -> String {
+    fn project_size_bytes(&self) -> u64 {
```

Comment:
When https://github.com/cloudflare/wrangler/pull/313 lands I think this logic can/should be moved into there so _all_ project types get a bundle size info message, not just wranglerjs/webpack projects.